File: Java/Data Structures & Algorithms Module - Java/Week1/Student.java
/*
Student.java
Billy Cussen
07/09/2020
*/

public class Student extends Person{

    //Constructor
    public Student(String firstName, String secondName, int age){
        super(firstName,secondName,age);
    }

    public String isStudent(){
        return "This is a Student";
    }

}

File: Python/Data Structures & Algorithms Module - Python/Week1/Student.py
"""
Student.py
Billy Cussen
09/11/2020
"""

import Person
from Person import myPerson

class myStudent(Person.myPerson):
    def __init__(self, firstName, secondName, age):
        super().__init__(firstName,secondName,age)

    def isStudent(self):
        return "This is a Student"